kubetest/anywhere.go

```diff
@@ -56,13 +56,16 @@ var (
 		"(kubernetes-anywhere only) Indicates whether to do the control plane upgrade with kubeadm method \"init\" or \"upgrade\"")
 	kubernetesAnywhereCNI = flag.String("kubernetes-anywhere-cni", "",
 		"(kubernetes-anywhere only) The name of the CNI plugin used for the cluster's SDN.")
+	kubernetesAnywhereProxymMode = flag.String("kubernetes-anywhere-proxy-mode", "",
+		"(kubernetes-anywhere only) Chose kube-proxy mode.")
 )
 
 const kubernetesAnywhereConfigTemplate = `
 .phase1.num_nodes={{.NumNodes}}
 .phase1.cluster_name="{{.Cluster}}"
 .phase1.ssh_user=""
 .phase1.cloud_provider="gce"
+.phase1.proxy_mode="{{.KubeproxyMode}}"
```

Comment:
I think this Config about `proxy_mode` is best suited in phase2.

Reply:
Yeah, you are right, and the feature not approved yet. I need to have a full test and make some change, push into community ASAP. Thanks!